cfme/tests/cloud/test_providers.py

```diff
@@ -478,10 +478,9 @@ def test_delete_arbitration_profiles_from_detail(self, appliance, arbitration_pr
         Metadata:
             test_flag: rest
         """
-        status = 204 if method == 'delete' else 200
         for entity in arbitration_profiles:
             entity.action.delete(force_method=method)
-            assert appliance.rest_api.response.status_code == status
+            assert appliance.rest_api.response.status_code in [200, 204]
```

Comment:
**optional** please use a tuple expression

Reply:
REQUIRED: please use ``assert_response(appliance)`` everywhere. This is legacy code that is being replaced by ``assert_response`` that correctly checks what status is expected.